pages/01-overview/03-api-reference.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: API Reference
+title: API Reference (Management)
```

Comment:
the script is almost certainly going to overwrite this. Might be worth getting another engineer to help out with refactoring it if we need this yaml injected.

Reply:
okay that should be fine for now, as long as it's getting updated properly and it exists in the docs.
we can quick follow to make sure title isn't overridden later